src/features/ExternalApi.ts

```diff
@@ -0,0 +1,144 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+import * as vscode from "vscode";
+import { v4 as uuidv4 } from 'uuid';
+import { LanguageClient } from "vscode-languageclient";
+import { IFeature } from "../feature";
+import { Logger } from "../logging";
+import { SessionManager } from "../session";
+
+interface IExternalExtension {
+    readonly id: string;
+    readonly apiVersion: string;
+}
```

Comment:
Should this be at the bottom, since it's not exported?

Reply:
moved